autotst/conformer/systematic.py

```diff
@@ -101,62 +101,16 @@ def find_all_combos(
     return all_combos
 
 
-def systematic_search(conformer,
-                      delta=float(120),
-                      energy_cutoff = 10.0, #kcal/mol
-                      rmsd_cutoff = 0.5, #angstroms
-                      cistrans = True,
-                      chiral_centers = True,
-                      multiplicity = False,
-                      ):
-    """
-    Perfoms a systematic conformer analysis of a `Conformer` or a `TS` object
-
-    Variables:
-    - conformer (`Conformer` or `TS`): a `Conformer` or `TS` object of interest
-    - delta (int or float): a number between 0 and 180 or how many conformers to generate per dihedral
-    - cistrans (bool): indication of if one wants to consider cistrans bonds
-    - chiral_centers (bool): indication of if one wants to consider chiral centers bonds
-
-    Returns:
-    - confs (list): a list of unique `Conformer` objects within 1 kcal/mol of the lowest energy conformer determined
-    """
-    
-    rmsd_cutoff_options = {
-        'loose' : 1.0,
-        'default': 0.5,
-        'tight': 0.1
-    }
-
-    energy_cutoff_options = {
-        'high' : 50.0,
-        'default' : 10.0,
-        'low' : 5.0
-    }
-
-    if isinstance(rmsd_cutoff,str):
-        rmsd_cutoff = rmsd_cutoff.lower()
-        assert rmsd_cutoff in rmsd_cutoff_options.keys(), 'rmsd_cutoff options are loose, default, and tight'
-        rmsd_cutoff = rmsd_cutoff_options[rmsd_cutoff]
-
-    if isinstance(energy_cutoff,str):
-        energy_cutoff = energy_cutoff.lower()
-        assert energy_cutoff in energy_cutoff_options.keys(), 'energy_cutoff options are low, default, and high'
-        energy_cutoff = energy_cutoff_options[energy_cutoff]
-    
-    if not isinstance(conformer, TS):
-        reference_mol = conformer.rmg_molecule.copy(deep=True)
-        reference_mol = reference_mol.to_single_bonds()
-    manager = multiprocessing.Manager()
-    return_dict = manager.dict()
-    pool = multiprocessing.Pool()
-
-    def opt_conf(i, rmsd_cutoff):
+def opt_conf(conformer):
         """
         A helper function to optimize the geometry of a conformer.
         Only for use within this parent function
         """
-        conformer = conformers[i]
+        #conformer = conformers[i]
+        print(conformer)
```

Comment:
Probably remove these, the comment isn't terribly helpful and the print statement doesn't do much either.